src/main/java/io/jenkins/plugins/analysis/core/model/ReportScanningTool.java

```diff
@@ -39,6 +39,7 @@
 
     private String pattern = StringUtils.EMPTY;
     private String reportEncoding = StringUtils.EMPTY;
+    private boolean followSymlinks = true;
```

Comment:
This helps only in pipelines. Actually we must use for new boolean fields always `false` as a default value. Otherwise it will be initialized in existing freestyle projects to `false`.  

So we need to inverse this field and rename it to `noFollowSymbolicLinks` or something similar (but no abbreviation of symbolic link like this is done in `Path` now)